core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2522,6 +2522,20 @@ public AlluxioURI getPath(long fileId) throws FileDoesNotExistException {
     return mInodeTree.getPinIdSet();
   }
 
+  @Override
+  public List<String> getPinnedFilePaths() {
+    Set<String> pinnedFiles = new HashSet<>();
+    for (long pinId : getPinIdList()) {
+      try {
+        String pinFilePath = getPath(pinId).getPath();
+        pinnedFiles.add(pinFilePath);
+      } catch (FileDoesNotExistException e) {
+        e.printStackTrace();
```

Comment:
instead of `e.printStackTrace();`, I would suggest `LOG.warn("Failed to find file for id {}: {}", pinId, e.getMessage())` to better work with Alluxio code convention.